### PR TITLE
Update github.com/google/cadvisor to tagged v0.36.0

### LINKED ISSUE #79499: hack/update-bazel produces warnings
**What happened**:
Got the following error while trying to run `hack/update-bazel.sh`:
```
gazelle: /home/user/go/src/k8s.io/kubernetes/vendor/github.com/seccomp/libseccomp-golang/seccomp.go: error reading go file: /home/user/go/src/k8s.io/kubernetes/vendor/github.com/seccomp/libseccomp-golang/seccomp.go: pkg-config not supported: #cgo pkg-config: libseccomp
gazelle: /home/user/go/src/k8s.io/kubernetes/vendor/github.com/seccomp/libseccomp-golang/seccomp_internal.go: error reading go file: /home/user/go/src/k8s.io/kubernetes/vendor/github.com/seccomp/libseccomp-golang/seccomp_internal.go: pkg-config not supported: #cgo pkg-config: libseccomp
``` 

**How to reproduce it (as minimally and precisely as possible)**:
Ran this on a fresh install of ubuntu 19.04, Go 1.12.6, and bazel 0.26.1.

**Anything else we need to know?**:
Seems like the change might have been due to the update in vendored deps from #78187 ( https://github.com/kubernetes/kubernetes/pull/78187#discussion_r294546736 ).
Although this script is run in CI where it is working so may be some set of recommende pkg versions would help prevent this problem?

**Environment**:
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`): ubuntu 19.04
- Kernel (e.g. `uname -a`): 5.0.0-15-generic
- bazel: 0.26.1
- Go: 1.12.6

### PR DESCRIPTION
**Pull request by aojea**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

Supress warning on the `hack/update-bazel.sh` script

**Which issue(s) this PR fixes**:

Fixes #79499

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: hack/update-bazel.sh [hunk 1]
```diff
@@ -70,7 +70,9 @@ gazelle fix \
     -mode=fix \
     -repo_root "${KUBE_ROOT}" \
     "${KUBE_ROOT}" \
-    2>&1 | grep -Ev "vendor/github.com/seccomp/libseccomp-golang/seccomp(_internal)?.go: pkg-config not supported: #cgo pkg-config: libseccomp" || (exit "${PIPESTATUS[0]}")
+    2>&1 | grep -Ev "vendor/github.com/seccomp/libseccomp-golang/seccomp(_internal)?.go: pkg-config not supported: #cgo pkg-config: libseccomp" \
```
Comment (tanjunchen) [inline hack/update-bazel.sh:73]: see https://github.com/kubernetes/kubernetes/pull/79502/files#diff-fa58360a7125fddab9491c80bb3f1a87R72

Comment (tanjunchen) [reply hack/update-bazel.sh:73]: @aojea  Thank  you. This output is a bit annoying.

https://github.com/kubernetes/kubernetes/issues/79499#issuecomment-614459419

**Review by BenTheElder — CHANGES_REQUESTED** [review]
Please see the previous PR chain. We did NOT just suppress the warnings.

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/kubernetes/pull/79287

**Comment by aojea** [comment]
/close

these modules dependencies are hard to me to deal with 🙃

**Pull request closed without merging**